src/common/src/array/stream_chunk.rs

```diff
@@ -70,14 +70,26 @@ impl Op {
 pub type Ops<'a> = &'a [Op];
 
 /// `StreamChunk` is used to pass data over the streaming pathway.
-#[derive(Default, Clone, PartialEq)]
+#[derive(Clone, PartialEq)]
 pub struct StreamChunk {
     // TODO: Optimize using bitmap
     ops: Vec<Op>,
 
     pub(super) data: DataChunk,
 }
 
+impl Default for StreamChunk {
+    /// Create a 0-row-0-col StreamChunk. Only used in some existing tests.
+    /// This is NOT the same as an **empty** chunk, which has 0 rows but with
+    /// columns aligned with executor schema.
+    fn default() -> Self {
```

Comment:
Rest LGTM!

Can we make two versions by #[cfg(test)] and #[cfg(not(test))], sort of worried people may abuse this.

Reply:
I think I might encounter a very similar problem, not of a method but of a module.